src/app/store/entity/entity.store.ts

```diff
@@ -64,6 +64,7 @@ export abstract class AbstractStore<T extends BaseEntity, L extends Array<T>,
         this._loading.next(false);
       },
       (error) => {
+        error = this.massageError(error);
```

Comment:
Wait, am I reading this correctly? Are we massaging errors now? Because I'm the one that really needs the massage here.. :neutral_face: